airflow/docker.go

```diff
@@ -347,6 +347,15 @@ func (d *DockerCompose) Pytest(pytestFile, projectImageName string) (string, err
 		return "", err
 	}
 
+	// Determine pytest file
+	if pytestFile != ".astro/test_dag_integrity_default.py" {
+		if !strings.Contains(pytestFile, "tests") {
+			pytestFile = "tests/" + pytestFile
```

Comment:
let's extract `tests` folder to constants.

Reply:
Okay I'll update thanks